br/pkg/conn/conn.go

```diff
@@ -1,4 +1,16 @@
-// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+// Copyright 2021 PingCAP, Inc.
```

Comment:
heh i thought the short boilerplate was also legal

Reply:
Make sense. Reverting...

BTW https://github.com/apache/skywalking/discussions/7756 and I'll check whether the short boilerplate is valid as the SPDX identifier is like

```
SPDX-License-Identifier: Apache-2.0
```

Reply:
Dealing with licenses for existing files later.

Reply:
Updated. Could you please help on merging this PR?